plugins/woocommerce/src/Internal/DownloadPermissionsAdjuster.php

```diff
@@ -43,6 +43,19 @@ public function maybe_schedule_adjust_download_permissions( \WC_Product $product
 			return;
 		}
 
+		$are_any_children_downloadable = false;
+		foreach ( $children_ids as $child_id ) {
+			$child = wc_get_product( $child_id );
+			if ( $child->is_downloadable() ) {
```

Comment:
I think you should check that `$child` is not false here.

Reply:
Thank you. Addressed in commit a97e883c08e059734972076b6708d7e529019011.